libbeat/processors/actions/rename.go

```diff
@@ -77,6 +77,7 @@ func (f *renameFields) Run(event *beat.Event) (*beat.Event, error) {
 		if err != nil && f.config.FailOnError {
 			logp.Debug("rename", "Failed to rename fields, revert to old event: %s", err)
 			event.Fields = backup
+			event.PutValue("error.message", fmt.Sprintf("Failed to rename fields in processor: %s", err))
```

Comment:
Nice addition. Perhaps we should also mention this in the changelog.

For the error, could you also create it one to use for the debug logging and event?

Reply:
@ruflin 	
> For the error, could you also create it one to use for the debug logging and event?

Can you clarify what you mean here?